README.md

```diff
@@ -0,0 +1,153 @@
+python-inspector - inspect Python packages dependencies and metadata
```

Comment:
We use ReST for docs, not markdown.  Why this change? Switching to markdown would mean changing the way we create and build documentation across ~ 100 repos including updating 1000 pages of docs. A big task that will unlikely happen unless this would bring an extraordinary value.

Reply:
Ok, i will revert to rst, no big